docusaurus.config.en.js

```diff
@@ -60,7 +60,7 @@ const config = {
   onBrokenLinks: "throw",
   onBrokenMarkdownLinks: "warn",
   onDuplicateRoutes: "throw",
-  onBrokenAnchors: process.env.ON_BROKEN_ANCHORS ?? "throw",
+  onBrokenAnchors: "warn",
```

Comment:
You can just set `ON_BROKEN_ANCHORS=true` locally, no need to change this. Please keep `onBrokenAnchors: process.env.ON_BROKEN_ANCHORS ?? "throw"`